stubs/lupa/lupa/lua51.pyi

```diff
@@ -0,0 +1,86 @@
+from _typeshed import MaybeNone
+from collections.abc import Callable, Iterator
+from typing import Any
+
+__all__ = [
+    "LUA_VERSION",
+    "LUA_MAXINTEGER",
+    "LUA_MININTEGER",
+    "LuaRuntime",
+    "LuaError",
+    "LuaSyntaxError",
+    "LuaMemoryError",
+    "as_itemgetter",
+    "as_attrgetter",
+    "lua_type",
+    "unpacks_lua_table",
+    "unpacks_lua_table_method",
+]
+
+LUA_MAXINTEGER: int
+LUA_MININTEGER: int
+LUA_VERSION: tuple[int, int]
+
+# cyfunction object
+as_attrgetter: Callable[[object], object]
+as_itemgetter: Callable[[object], object]
+
+# cyfunction object
+lua_type: Callable[[object], str | MaybeNone]
+
+# cyfunction object as decorator
+unpacks_lua_table: Callable[[Callable[..., Any]], Callable[..., Any]]
+unpacks_lua_table_method: Callable[[Callable[..., Any]], Callable[..., Any]]
+
+# inner classes
+
+class _LuaIter:
+    def __iter__(self) -> Iterator[object]: ...
+
+class _LuaTable:
+    def keys(self) -> _LuaIter: ...
+    def values(self) -> _LuaIter: ...
+    def items(self) -> _LuaIter: ...
+
+class _LuaNoGC: ...
+class _LuaObject: ...
```

Comment:
I these are not available at runtime, they should be annotated with `@type_check_only` (from `typing`). Same below.